massmailer/models.py

```diff
@@ -189,45 +196,11 @@ def parse(self):
 
     @staticmethod
     def execute(query):
-        User = get_user_model()
```

Comment:
Looks like you didn't clean the imports (I guess this is isn't used anywhere else?)